rosidl_typesupport_connext_c/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 2.8.3)
+
+project(rosidl_typesupport_connext_c)
+
+if(NOT WIN32)
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c11 -Wall -Wextra")
```

Comment:
Since the `identifier.cpp` file is built by this package, I think you need to set the `CMAKE_CXX_FLAGS` as well. Also I'd recommend dropping the `-std=c11` flag, since we're not using anything that is C11 specific right now.